tests/featureWriters/kernFeatureWriter_test.py

```diff
@@ -1840,6 +1840,76 @@ def test_kern_hira_kana_hrkt(FontClass):
     )
 
 
+# flake8: noqa: B950
+def test_redefining_class_defs(FontClass, datadir):
+    """Test that class definitions aren't overwritten by either the same content
```

Comment:
"overwritten with the same content" is usually benign.. it's the silently redefined which is the problem. But by just reading below, it isn't exactly clear to me where this redefinition occurs